src/components/application/List.jsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Card,
+  CardBody,
+  CardFooter,
+  Flex,
+  Heading,
+  Text,
+  Center,
+  Badge,
+  UnorderedList,
+  ListItem,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  Divider,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+} from "@chakra-ui/react";
+import '../../assets/styles/App.css'
+import { ArrowBackIcon, ArrowForwardIcon } from "@chakra-ui/icons";
+import { Link } from "react-router-dom";
+import Header from "../common/Header";
+import Footer from "../common/Footer";
+
+const List: React.FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const finalRef = React.useRef(null);
+
+  const openModal = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault(); 
+    onOpen(); 
+  };
+  return (
+    <Box className="main-bg">
```

Comment:
Please add a common layout component.
Don't use className or CSS classes here; use components and props instead.

Reply:
done